libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -57,8 +57,13 @@ smt::SortPointer dev::solidity::smtSort(Type const& _type)
 			solAssert(mapType, "");
 			return make_shared<smt::ArraySort>(smtSort(*mapType->keyType()), smtSort(*mapType->valueType()));
 		}
-		// TODO Solidity array
-		return make_shared<smt::Sort>(smt::Kind::Int);
+		else
+		{
+			solAssert(isArray(_type.category()), "");
```

Comment:
Is it normal to have empty messages for the asserts?

Reply:
Yes. They shouldn't fail anyway, and if they do, it looks like a crash to the user so it doesn't really matter (in my opinion). As long as they report it, we can easily find the broken assertion.